From: dnf-for-termux <dev@termux.dev>
Date: Fri, 31 Jul 2026 00:00:00 -0600
Subject: [PATCH 10/10] Disable needs_restarting_plugin (requires sdbus-c++)

The needs_restarting_plugin includes cmake/sdbus_cpp.cmake which
declares pkg_check_modules(SDBUS_CPP REQUIRED sdbus-c++>=1).
sdbus-c++ is not packaged in termux-packages, so building this
plugin would fail at configure time. There is no cmake option to
disable it individually, so skip the add_subdirectory call.

diff --git a/dnf5-plugins/CMakeLists.txt b/dnf5-plugins/CMakeLists.txt
--- a/dnf5-plugins/CMakeLists.txt
+++ b/dnf5-plugins/CMakeLists.txt
@@ -15,5 +15,5 @@
 add_subdirectory("changelog_plugin")
 add_subdirectory("config-manager_plugin")
 add_subdirectory("copr_plugin")
-add_subdirectory("needs_restarting_plugin")
+# needs_restarting_plugin disabled for Termux: requires sdbus-c++ (not packaged)
 add_subdirectory("repoclosure_plugin")
 add_subdirectory("reposync_plugin")
 add_subdirectory("repomanage_plugin")
